### PR TITLE
Capabilities v2

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**

### CODE REVIEW

File: blocks/kmeans_clustering/UP42Manifest.json [hunk 1]
```diff
@@ -33,9 +33,19 @@
...
+            "up42_standard": {
+                "format": "GTiff",
+            }
+        }
     },
     "output_capabilities": {
-        "up42.data.aoiclipped": {}
+        "raster": {
+            "up42_standard": {
+                "format": "GTiff"
```
Comment (sandor-nemeth-up42) [inline blocks/kmeans_clustering/UP42Manifest.json]: trailing comma is missing

File: blocks/kmeans_clustering/UP42Manifest.json [hunk 2]
```diff
@@ -33,9 +33,19 @@
 	    "type": "xlarge"
     },
     "input_capabilities": {
-        "up42.data.aoiclipped": {}
+        "raster": {
+            "up42_standard": {
+                "format": "GTiff",
```
Comment (sandor-nemeth-up42) [inline blocks/kmeans_clustering/UP42Manifest.json]: extra trailing comma

Comment (rodrigoalmeida94) [reply blocks/kmeans_clustering/UP42Manifest.json]: Fixed!

File: blocks/kmeans_clustering/UP42Manifest.json (hunk 1 above)
Comment (rodrigoalmeida94) [reply blocks/kmeans_clustering/UP42Manifest.json]: Thanks!

**Pull request merged**